### ISSUE TITLE
Error using callable Class as relation_scope

### ISSUE THREAD

**Issue by brendon** [palkan/action_policy#268]
### Tell us about your environment

**Ruby Version: 3.0.56

**Framework Version (Rails, whatever): 6.1

**Action Policy Version: 0.6.9

### What did you do?

Following the documentation around a callable relation_scope:

```
class PostsController < ApplicationController
  def index
    @posts = authorized_scope(Post.all)
  end
end

class PostPolicy < ApplicationPolicy
  relation_scope AuthorizedPosts
end

class AuthorizedPosts
  class << self
    def call(policy, relation)
      user = policy.user
      user.admin? ? relation : relation.where(user: user)
    end
  end
end
```

### What did you expect to happen?

It should call the class `call` method.

### What actually happened?

I get the error message `tried to create Proc object without a block` on `action_policy (0.6.9) lib/action_policy/policy/scoping.rb line 119`

I can create a reproduction script but just wanted to check first in case you can see the problem already :)

**Comment by palkan** [comment]
Oh, this feature hasn't been released yet. I've just published v0.7.0 which includes it. Please, check, and feel free to re-open if there are still issues.

**Comment by brendon** [comment]
Haha! Bleeding edge eh!

I ended up using a concern as this felt a little more natural. Perhaps less readable though. Will give this another try.

**Comment by brendon** [comment]
It works well. It does feel a bit weird not having an official way to name and store these classes though. Did you have any thoughts? Right now I've namespaced it to the particular module in my app along with `AuthorisedResources`: `MyModule::AuthorisedResources` and have stored it in `policies/my_module/authorised_resources.rb`.

**Comment by palkan** [comment]
> It does feel a bit weird not having an official way to name and store these classes though

Well, that's a very recent community-driven addition, not something taken into account by the initial library design.

I would put scope classes under the policy class namespace and name them as `XXXScope`, e.g.:

```sh
policies/
  post_policy/
    posts_scope.rb
  post_policy.rb
```

**Comment by brendon** [comment]
Hi @palkan, just following this up quickly. Does a callable class still support passing `scope_options`? If so, I'm having trouble figuring out how. Do you have any tips?

**Comment by palkan** [comment]
@brendon Thanks for catching this! We forgot about scope options. Fixed and released in 0.7.1 (see [this commit](https://github.com/palkan/action_policy/commit/ecc6a0ce1e90453aaee4503fb04fa6ca07d22eee) for examples).

**Comment by brendon** [comment]
Nice! :D Glad to help :)
